### DISCUSSION TITLE
multiple config

### DISCUSSION THREAD

**Discussion by bhbmalka** [aahnik/tgcf#250] [Q&A]
Can we deploy 1 tgcf and give several config?

**Comment by aahnik** [comment]
Your question is not clear to me.

Please read the full readme, and all the pages of wiki.

What do you mean by multiple config ?

You can run one instance of tgcf with multiple source and destinations. Read https://github.com/aahnik/tgcf/discussions/68

You may also run multiple instances of tgcf in one machine. Read https://github.com/aahnik/tgcf/discussions/248

**Reply by bhbmalka** [reply]
like this

forwards:
  - source: -1001412118156
    dest: [-1001461332656, -1001457230530]
  - source: -1003456789
    dest: [-13456789067]



can i use different filter on that 2 multi source-dest

**Reply by bhbmalka** [reply]
and is it possible for only forward message with several line foe avoid spam ( make bot not forward to long message)

**Reply by tissole** [reply]
Currently is not possible to use separate plugins per source, but is planned for the future. See #176.

Messages regardless of their length/lines are still considered as one message.
